DESCRIPTION

```diff
@@ -39,10 +39,10 @@ Imports:
     ggplot2,
     lifecycle (>= 0.2.0),
     magrittr,
-    mmrm (>= 0.3.5),
+    mmrm (>= 0.3.13),
```

Comment:
I would keep this at the current minimum version requirement, no need to upgrade (and don't upgrade to 0.3.13 because that version is not yet on CRAN)